test/functional/wallet_migration.py

```diff
@@ -445,6 +445,39 @@ def test_no_privkeys(self):
         # After migrating, the "keypool" is empty
         assert_raises_rpc_error(-4, "Error: This wallet has no available keys", watchonly1.getnewaddress)
 
+        # Migrating a watch-only blank empty wallet (with no pubkeys)
+        self.log.info("Test migration of a pure watchonly blank empty wallet (with no pubkeys)")
+        watchonly2 = self.create_legacy_wallet("watchonly2", disable_private_keys=True, blank=True)
+        # Before migrating
+        info = watchonly2.getwalletinfo()
+        assert_equal(info["descriptors"], False)
+        assert_equal(info["private_keys_enabled"], False)
+        assert_equal(info["blank"], True)
+
+        _, watchonly2 = self.migrate_and_get_rpc("watchonly2")
+        # After migrating
+        info = watchonly2.getwalletinfo()
+        assert_equal(info["descriptors"], True)
+        assert_equal(info["private_keys_enabled"], False)
+        assert_equal(info["blank"], True)
```

Comment:
In https://github.com/bitcoin/bitcoin/commit/9962614b9060e9713042de91dc04ff5ea6604ae1:

This is actually asserting the wrong behavior. The migrated wallet should have private keys disabled, just like the legacy wallet.

Reply:
ha, I read it in the other way around.. "private_keys_disabled".. all good. Can solve this comment.